ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -481,7 +481,7 @@ func resourceIBMPIInstanceRead(ctx context.Context, d *schema.ResourceData, meta
 		d.Set("min_virtual_cores", powervmdata.VirtualCores.Min)
 	}
 	d.Set(helpers.PIInstanceLicenseRepositoryCapacity, powervmdata.LicenseRepositoryCapacity)
-
+	d.Set(PIInstanceDeploymentType, powervmdata.DeploymentType)
```

Comment:
If user don't pass any deploymentType during create what does the GET return back ?

Reply:
returns empty